### PR TITLE
Added switch to disable dragging map marker

### PR DESCRIPTION
**Pull request by sfk-steelsong**
## Description

Added a switch (client side) to enable / disable dragging of the marker on the map.  It defaults to disabled if the device has touch enabled.

## Motivation and Context

It seemed too easy to accidentally drag the marker when moving the map, particularly on mobile, which would restart scanning.

## How Has This Been Tested?

Desktop:
Windows 7 - IE  11, Chrome 52, Firefox 48
Ubuntu 16.04 - Chrome 54

Mobile:
iPhone - iOS 9.3.2: Chrome 52, Safari,
Android 7: Chrome 52 mobile

## Screenshots (if appropriate):
marker

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: static/js/map.js [hunk 1]
```diff
@@ -1734,6 +1739,11 @@ function i8ln (word) {
   }
 }
 
+function isTouchDevice () {
+  return 'ontouchstart' in window ||  // works on most browsers
+  navigator.maxTouchPoints;  // works on IE10/11 and Surface
```
Comment (chuyskywalker) [inline static/js/map.js]: Eeeeee, I get that this works as javascript, but put it all on one line. Ick.

**Comment by chuyskywalker** [comment]
Other than a small JS issue, this LGTM

File: static/js/map.js (hunk 1 above)
Comment (sfk-steelsong) [inline static/js/map.js]: Yeah, only did it because lint yelled at me. Fixing!

**Comment by sfk-steelsong** [comment]
cleaned up the formatting.

**Comment by invisiblek** [comment]
I never liked this feature to begin with tbh...

**Comment by sfk-steelsong** [comment]
@invisiblek Any particular reason why you dislike this? I'm open to making changes...

**Comment by chuyskywalker** [comment]
(I believe he means to say "This is a good add, I hate the fact that you can accidentally move the scan location")

**Comment by invisiblek** [comment]
Yea, that's what I mean. I'd personally rather rip this all out completely, but that's just me =)

+1 on this

**Comment by sfk-steelsong** [comment]
Haha, fair.  I don't like to make assumptions before having caffeine...

**Pull request merged**